com.sap.cloud.lm.sl.cf.process/src/main/java/com/sap/cloud/lm/sl/cf/process/steps/DetectApplicationsToRenameStep.java

```diff
@@ -0,0 +1,122 @@
+package com.sap.cloud.lm.sl.cf.process.steps;
+
+import com.sap.cloud.lm.sl.cf.core.model.BlueGreenApplicationNameSuffix;
+import com.sap.cloud.lm.sl.cf.core.model.DeployedMta;
+import com.sap.cloud.lm.sl.cf.core.model.DeployedMtaApplication;
+import com.sap.cloud.lm.sl.cf.core.util.CloudModelBuilderUtil;
+import com.sap.cloud.lm.sl.cf.process.message.Messages;
+import org.cloudfoundry.client.lib.domain.CloudApplication;
+import org.cloudfoundry.client.lib.domain.ImmutableCloudApplication;
+import org.flowable.engine.delegate.DelegateExecution;
+import org.springframework.beans.factory.config.BeanDefinition;
+import org.springframework.context.annotation.Scope;
+
+import javax.inject.Named;
+import java.util.Collections;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@Named("detectApplicationsToRenameStep")
+@Scope(BeanDefinition.SCOPE_PROTOTYPE)
+public class DetectApplicationsToRenameStep extends SyncFlowableStep {
+
+    private static final String APP_NAME_OLD_SUFFIX = BlueGreenApplicationNameSuffix.OLD.asSuffix();
```

Comment:
Remove this and replace it with the already existing one in the BlueGreenApplicationNameSuffix

Reply:
Done